lib/wrappers/rust/Readme.md

```diff
@@ -0,0 +1,26 @@
+# Rust wrapper 
+
+The rust wrapper library wrapping the cw-core library, which is used to control the contour wall. The examples of using rust wrapper is put at the end of the lib.rs file.
+
+# How to run
+
+1. Install Cargo Rust version >=  1.75
+2. Install Rust wrapper dependencies:
+```
+cargo add ndarray
+cargo add serialport@=4.3.0
+```
+3. Set the library type of cw-core to ‘dylib’ with the following configuration in cargo.toml
+```
+[lib]
+crate-type = ["dylib"]
```

Comment:
This is unnecessary, as all of these things are already defined in the `Cargo.toml` files, these steps could be removed